src/admin/components/ResourceTabs/InformationTab/index.tsx

```diff
@@ -114,10 +126,30 @@ const InformationTab: FC<IProps> = props => {
                   </Card>
 
                   <When condition={record}>
+                    <When
+                      condition={
+                        record.framework_key === "ppc" && (props.type === "sites" || props.type === "site-reports")
```

Comment:
you could use props.type.startsWith('site')

Reply:
also for the `record.framework_key === "ppc"` please use `useFramework` custom hook.